pkg/api/v1beta3/dynakube/extensions.go

```diff
@@ -102,3 +103,11 @@ type ImageRefSpec struct {
 	// Indicates a tag of the image to use
 	Tag string `json:"tag,omitempty"`
 }
+
+func (dk *DynaKube) GetExtensionsTLSSecretName() string {
```

Comment:
shouldn't this be in the `extensions_props.go` ?

Reply:
Fixed!